linux/snap/snapcraft.yaml

```diff
@@ -0,0 +1,75 @@
+name: nextcloud-client
+version: 2.2.4-git
+summary: Nextcloud Desktop Client
+icon: ../nextcloudtheme/theme/colored/Nextcloud-icon.svg
+description: |
+  The Nextcloud Desktop Client is a tool to synchronize files from Nextcloud
+  Server with your computer.
+
+grade: devel
+confinement: strict
+
+parts:
+  client:
+    plugin: cmake
+    source: ../
+    source-subdir: client
+    build-packages:
+      - g++
+      - libqt5webkit5-dev
+      - libsqlite3-dev
+      - libssl-dev
+      - pkg-config
+      - qt5keychain-dev
+      - qttools5-dev-tools
+    configflags:
+      - -DOEM_THEME_DIR=$SNAPCRAFT_PART_INSTALL/../src/nextcloudtheme
+
+      # XXX: This is an hack to have a kind of bind-mount with absolute prefix.
+      - -DCMAKE_INSTALL_PREFIX=/snap/$SNAPCRAFT_PROJECT_NAME/current/usr
+      - -DCMAKE_INSTALL_LIBDIR=/snap/$SNAPCRAFT_PROJECT_NAME/current/usr/lib
+      - -DSYSCONF_INSTALL_DIR=/snap/$SNAPCRAFT_PROJECT_NAME/current/etc
```

Comment:
You say you get a log denying access. Do you still get that message? Do you get that message with nextcloud-client-kyrofa?